### ISSUE TITLE
External blobstore (S3) can't be used

### ISSUE THREAD

**Issue by phil9909** [cloudfoundry-incubator/eirini-staging#11]
eirini-staging does not trust "common" CAs. For example not the "Baltimore CyberTrust Root" which is currently used by AWS S3.

```bash
error installing: failed to perform get request on: https://apps-hyperion.s3-eu-central-1.amazonaws.com/e8/45/e845748[...]: Get https://apps-hyperion.s3-eu-central-1.amazonaws.com/e8/45/e8457[...]: x509: certificate signed by unknown authority
```

We tried to fix this with #9 which has unresolvable conflicts by now. Since `cloudfoundry.org/tlsconfig` is used now the fix is no longer simple.

@alex-slynko proposed opening an issue at `cloudfoundry.org/tlsconfig`, but we think `WithAuthorityFromFile` works as designed:

https://github.com/cloudfoundry-incubator/eirini-staging/blob/4c298819d2a4529b3fd9169d245e2069c1f954a7/vendor/code.cloudfoundry.org/tlsconfig/config.go#L173-L175

Maybe we need something like `WithAdditionalAuthorityFromFile`.

@julz since you gave a 👍on https://cloudfoundry.slack.com/archives/C8RU3BZ26/p1573564475120400 I assume you agree that S3 should be supported by eirini-staging. Correct me if I'm wrong.

**Comment by julz** [comment]
I agree in theory that we'd like to support S3 as a blob store. We're also a little puzzled by the right way to do it to be honest, given ideally we'd like to use `cloundfoundry.org/tlsconfig` :-). Would you be willing to try a draft PR for a fix based on that? Or we can pull in a story to our backlog to look at it (probably in the new year, now, though).

**Comment by jkbschmid** [comment]
We updated our previous PR to make use of `cloundfoundry.org/tlsconfig`. We will also open a PR for `cloundfoundry.org/tlsconfig` to implement the additional functions we need.

**Comment by alex-slynko** [comment]
fixed by #9
